conf/email.routes

```diff
@@ -2,3 +2,4 @@ POST        /insert                          @controllers.EmailController.insert
 DELETE      /remove                          @controllers.EmailController.remove
 GET         /count                           @controllers.EmailController.count
 
+POST        /                                @controllers.EmailController.insertWithResponse
```

Comment:
As above, we don't need two insert routes so maybe just amend the existing `/insert` to be `/`